Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3-stretch
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+      python3-dev libxml2-dev libxslt1-dev libffi-dev git glpk-utils && \
+    rm -rf /var/lib/apt/lists/* && \
+    pip3 install pipenv
+
+ADD . /app
```

Comment:
This isn't strictly necessary in development, as `/app` is mounted as a volume, and it'll slow the build time down as well as generate more intermediate docker images locally.

I suggest we remove this, then we can add a different Dockerfile for production when the time comes.

(Strictly it would also be [stylistically better](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy) to use `COPY` rather than `ADD` here.)

Reply:
Switched that to copy only the files needed to bootstrap the virtualenv.